src/lib/math/bigint/bigint.h

```diff
@@ -20,6 +20,8 @@ namespace Botan {
 
 class RandomNumberGenerator;
 
+static constexpr size_t BOTAN_MP_WORD_BITS = sizeof(word) * 8;
```

Comment:
SemVer nit: This is now much narrower in scope, i.e. only available when users include `bigint.h`. Is that something we want to live with?

Perhaps even deprecate it?

Reply:
I think it is best to just remove it right away. It is very unclear what exactly someone would be using it for, and we only documented it as a toggle to control the size of `word` (and probably roughly zero people ever did that.) I checked rnp, KeepassXC, and SoftHSM, none use it. No hits in Google search or Github code search either, except of course us.

Remove it in this PR gives this the same treatment as the other macros previously in build.h; they just vanish, as far as the user is concerned.

Maybe 3.8 comes out and someone complains. More likely nobody ever notices.